### PR TITLE
chore: change searchbyIngs icon on home screen/added small mod for ta…

### PR DESCRIPTION
**Pull request by stefanomusicman**
…ste screen

# Description of the problem
changed search by ingredients icon on home screen

## How does it solve the problem
BAR-99

# References (also the Jira ticket)

# Checklist:

- [x] I re-reviewed the Jira ticket now to make sure 
- [x] My code follows the style guidelines of this project & of Dart
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [x] I have wrote some documentation if I changed a big behavior of the code
- [x] My changes generate no new warnings/and if they do, I described them to the team
- [ ] I have added tests that prove my fix is effective or that my feature works if I worked on a logic class/function
- [ ] New and existing unit and integration tests pass locally with my changes

### CODE REVIEW

**Comment by stefanomusicman** [comment]
i feel the search by ingredients icon is a little small but if i increase it anymore the others will be too big

**Comment by stefanomusicman** [comment]
Untitled

File: lib/ui/screens/searchByTaste/search_by_taste_main.dart [hunk 1]
```diff
@@ -11,6 +12,7 @@ class SearchByTasteMain extends StatelessWidget {
   Widget build(BuildContext context) {
     Size size = MediaQuery.of(context).size;
     void navigateToHome() {
+      filters.clear();
```
Comment (danielemonaca) [inline lib/ui/screens/searchByTaste/search_by_taste_main.dart:15]: nice!

File: lib/ui/screens/home/discover_new_ones.dart [hunk 2]
```diff
@@ -44,6 +44,8 @@ class DiscoverNewOnes extends StatelessWidget {
                       title: item.title,
                       image: item.image,
                       action: () => item.action,
+                      height: 0.1,
+                      width: item.title == 'Search by Ingredient' ? 0.2 : 0.15,
```
Comment (danielemonaca) [inline lib/ui/screens/home/discover_new_ones.dart:48]: What happens if there is not this one? 
Is the `svg` wrong in any `anyway` that I could adjust?

**Review by danielemonaca — APPROVED** [review]

**Pull request merged**